CMakeLists.txt

```diff
@@ -61,7 +61,11 @@ else()
 
     add_definitions(-fPIC)
     if(CMAKE_BUILD_TYPE MATCHES "(Release|RELEASE|release)")
-        add_definitions(-Ofast)
+        if(NOT CMAKE_SYSTEM_NAME STREQUAL "Emscripten")
+            add_definitions(-Ofast)
+        else()
+            add_definitions(-O3)
```

Comment:
-O3 is the default option with CMAKE_BUILD_TYPE=release
shall be unnecessary here